PebbleSharp.Core/Install/InstallClient.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Collections.Generic;
+using PebbleSharp.Core.Responses;
+using PebbleSharp.Core.Bundles;
+using PebbleSharp.Core.BlobDB;
+
+namespace PebbleSharp.Core.Install
+{
+	public class InstallClient
+	{
+		private Pebble _pebble;
+		public InstallClient(Pebble pebble)
+		{
+			_pebble = pebble;
+		}
+		public async Task InstallAppAsync( AppBundle bundle, IProgress<ProgressValue> progress = null )
+		{
+
+			string version = _pebble.Firmware.Version;
+			version = version.Replace("v", "");
+			var components = version.Split(new char[] { '.','-' }, StringSplitOptions.RemoveEmptyEntries);
+
+			int i;
+			IList<int> versionComponents = components.Where(x=>int.TryParse(x,out i)).Select(x => int.Parse(x)).ToList();
```

Comment:
I also decided to move it to FirmwareVersion, that seems like a more sensible home for it.  It seems like there is a good chance we'll be doing this sort of check a lot in the future as services are added/removed in pebble revs.